### PR TITLE
Add more mac group tests

### PR DESCRIPTION
**Pull request by Ch3LL**
### What does this PR do?

Adds additional tests for the mac group module
### What issues does this PR fix or reference?

na
### Previous Behavior

there were no tests availabes for the following modules: group.adduser, group.deluser, group.members, group.getent
### New Behavior

Now there are tests for the following modules: group.adduser, group.deluser, group.members, group.getent
### Tests written?
- [x] Yes
- [ ] No

### CODE REVIEW

File: tests/integration/modules/mac_group.py [hunk 1]
```diff
@@ -114,6 +115,88 @@ def test_mac_group_chgid(self, grains=None):
...
+        # Create a group to use for test - If unsuccessful, skip the test
+        if self.run_function('group.add', [ADD_GROUP, 5678]) is not True:
+            self.run_function('group.delete', [ADD_GROUP])
+            self.skipTest('Failed to create a group to manipulate')
+
+        try:
+            self.run_function('group.adduser', [ADD_GROUP, ADD_USER])
+            group_info = self.run_function('group.info', [ADD_GROUP])
+            self.assertEqual(ADD_USER, ''.join(group_info['members']))
+        except AssertionError:
```
Comment (cachedout) [inline tests/integration/modules/mac_group.py]: A better pattern here is to use `setUp` and `tearDown`. See https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUp

File: tests/integration/modules/mac_group.py [hunk 2]
```diff
@@ -114,6 +115,88 @@ def test_mac_group_chgid(self, grains=None):
...
+    @destructiveTest
+    @skipIf(os.getuid() != 0, 'You must be logged in as root to run this test')
+    @requires_system_grains
+    def test_mac_deluser(self, grains=None):
+        '''
+        Test deleting user from a group
+        '''
+        # Create a group to use for test - If unsuccessful, skip the test
+        if self.run_function('group.add', [ADD_GROUP, 5678]) and \
+           self.run_function('group.adduser', [ADD_GROUP, ADD_USER])is not True:
```
Comment (cachedout) [inline tests/integration/modules/mac_group.py]: Missing a space?

**Comment by cachedout** [comment]
This is pretty good, but I think the `setUp` and `tearDown` approach might be more robust verus all these conditionals. Let me know what you think.

**Comment by rallytime** [comment]
These changes were merged in with #32028.

**Pull request closed without merging**